sale_request/i18n/es.po

```diff
@@ -880,19 +887,26 @@ msgstr "No puede solicitar más cantidad que la demanda inicial de esta requisic
 
 #. module: sale_request
 #: code:addons/sale_request/wizards/create_sale_order_wizard.py:222
-#: code:addons/sale_request/wizards/create_sale_order_wizard.py:360
+#: code:addons/sale_request/wizards/create_sale_order_wizard.py:363
 #, python-format
 msgid "You cannot request more than the remaining qty of this master sale order."
 msgstr "No puedes requerir más que la cantidad restante de este pedido master."
 
+#. module: sale_request
+#: code:addons/sale_request/models/sale_request.py:394
+#, python-format
+msgid "You do not have enougth stock of these components:\n"
+" %s"
+msgstr "No tienes sufieciente stock en los siguientes componentes:\n"
+
```

Comment:
Don't remove the %s at the end it will lead to an error,

also check te typo in `sufieciente` it must be `suficiente`